### DISCUSSION TITLE
[之前求助🆘不小心关闭]蓝灯充值2年，频繁连接不上，只刚开始使用三个小时，能否退款？ #478

### DISCUSSION THREAD

**Discussion by zy0628zy** [getlantern/lantern-forum-cn#484] [Q&A]
![Screenshot_2024-05-02-21-46-54-73_439a3fec0400f8974d35eed09a31f914](https://github.com/getlantern/lantern-forum-cn/assets/168475629/fd091ab4-3d95-4903-a8b4-a091b41d9d03)
![Screenshot_2024-05-02-21-46-45-99_439a3fec0400f8974d35eed09a31f914](https://github.com/getlantern/lantern-forum-cn/assets/168475629/9c98ef55-6c61-4e71-b6b1-e0bca5b7e0ec)
![Screenshot_2024-05-02-21-50-32-44](https://github.com/getlantern/lantern-forum-cn/assets/168475629/a36a38e1-8dd3-436c-bf12-3ce077491768)

**Comment by zy0628zy** [comment]
北京时间5月2日登录失败记录
![Screenshot_2024-05-02-21-39-07-55_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/ea397f4d-1e44-4145-aa66-8506357fc826)
![Screenshot_2024-05-02-10-54-05-87_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/2b860f44-25e7-4f77-845e-837a88f80cc7)
![Screenshot_2024-05-02-10-44-38-00_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/7ec26037-9250-4d77-8f45-44d148bc1994)
![Screenshot_2024-05-02-00-19-39-00_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/28df0137-8be7-49c5-9ef8-69fce6d57e37)

**Comment by zy0628zy** [comment]
5月1日登录失败记录
![Screenshot_2024-05-01-16-15-22-44_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/80a574ce-f947-47cb-b534-0a9ae54173c8)
![Screenshot_2024-05-01-13-37-32-86_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/892e0a71-d391-4e2f-bf27-915f4b6c4d5d)

**Comment by zy0628zy** [comment]
4月30日登录失败记录
![Screenshot_2024-04-30-21-05-16-07_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/13ded9fd-cee3-48af-b8b4-40848ada5317)
![Screenshot_2024-04-30-17-44-12-40_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/8c9379ce-de4e-4d6d-ad9b-88ded49eac0f)
![Uploading Screenshot_2024-04-30-17-43-38-32_4b7dd4d3471a9bd5811f20f1c9979b1f.jpg…]()

**Comment by zy0628zy** [comment]
所以你们答应我可以退款，什么时候可以退给我？
我的邮箱
zhangying06283027@126.com
付款记录
![Screenshot_2024-04-30-03-13-12-61_cf3cf72bd8e53b0db7ddb0a6f2208af9](https://github.com/getlantern/lantern-forum-cn/assets/168475629/6ad7b677-cf9c-4125-aead-5efa8b54a600)

![Screenshot_2024-04-30-00-24-27-28_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/2dd08423-e696-42a6-879c-28237fee0201)

**Comment by zy0628zy** [comment]
才使用四天，就频繁不能登陆。撞大运一样难得上论坛。我和你们的论坛管理人员回复，不愿意再给你们机会，我没一天能顺畅使用。你们的客服人员也答应可以🉑退款。请尽快把钱💰退给我谢谢。**

**Comment by zy0628zy** [comment]
我还有很多连接错误❌的截图我就不一一贴图了，请论坛管理人员联系支付团队快点退款，谢谢。

**Comment by zy0628zy** [comment]
无论换美国🇺🇸还是新加坡🇸🇬服务器都不行，日本🇯🇵服务器就没见过。
![Screenshot_2024-05-02-21-39-07-55_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/fc3d4132-7d47-4844-8caa-54ecdc2da1f3)
![Screenshot_2024-04-30-20-05-33-47_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/d7da5c72-1661-4144-8fb8-a917f768024b)
麻烦给我退款吧，实在无法忍受。我都快疯了。

**Reply by MaxMKMax** [reply]
@zy0628zy 您好. 已经帮您提交退款申请给负责部门了. 请您等待 14-20 天的退款流程. 请您放心. 谢谢.
